packages/terra-switch/src/Switch.module.scss

```diff
@@ -0,0 +1,199 @@
+
+:local {
+  .switch {
+    background-color: transparent;
+    border: 0;
+    display: flex;
+    flex-direction: row;
+
+    &:hover {
+      background-color: transparent;
+      border: 0;
+
+      .slider:not(.is-disabled) {
+        background-color: var(--terra-switch-slider-hover-background-color, #fff);
+        border: var(--terra-switch-slider-hover-border, 0);
+        box-shadow: var(--terra-switch-slider-hover-box-shadow, 2px 0 2px 0 rgba(0, 0, 0, 0.2), -2px 0 2px 0 rgba(0, 0, 0, 0.2), 
+        1px 0 0 0 #909496, 0 0 4px 1px rgba(0, 0, 0, 0.2));
+      }
+
+      .slider.is-selected:not(.is-disabled) {
+        background-color: var(--terra-switch-slider-selected-hover-background-color, #fff);
+        border: var(--terra-switch-slider-selected-hover-border, 0);
+        box-shadow: var(--terra-switch-slider-selected-hover-box-shadow, 2px 0 2px 0 rgba(0, 0, 0, 0.2), -2px 0 2px 0 rgba(0, 0, 0, 0.2), 
+        -1px 0 0 0 #01639e, 0 0 4px 1px rgba(0, 0, 0, 0.2));
+      }
+    }
+
+    &:active {
+      .slider:not(.is-disabled) {
+        background-color: var(--terra-switch-slider-active-background-color, #fff);
+        border: var(--terra-switch-slider-active-border, 0);
+        box-shadow: var(--terra-switch-slider-active-box-shadow, inset 0 0 0 1px #fff, inset 5px 2px 4px 2px #e6e7e8, 2px 0 2px 0 rgba(0, 0, 0, 0.2),
+        -2px 0 2px 0 rgba(0, 0, 0, 0.2), 1px 0 0 0 #909496, 0 0 4px 1px rgba(0, 0, 0, 0.2)); // pressed state.
+      }
+
+      .slider.is-selected:not(.is-disabled) {
+        background-color: var(--terra-switch-slider-selected-active-background-color, #fff);
+        border: var(--terra-switch-slider-selected-active-border, 0);
+        box-shadow: var(--terra-switch-slider-selected-active-box-shadow, inset 0 0 0 1px #fff, inset 5px 2px 4px 2px #e6e7e8, 2px 0 2px 0 rgba(0, 0, 0, 0.2),
+        -2px 0 2px 0 rgba(0, 0, 0, 0.2), -1px 0 0 0 #01639e, 0 0 4px 1px rgba(0, 0, 0, 0.2)); // pressed state.
+      }
+    }
+
+    &:focus {
+      background-color: transparent;
+      border: 0;
+      outline: 0;
+
+      &[data-terra-switch-show-focus-styles='true'] .slider:not(.is-disabled) {
+        background-color: var(--terra-switch-slider-focus-background-color, #fff);
+        border: var(--terra-switch-slider-focus-border, 0);
+        box-shadow: var(--terra-switch-slider-focus-box-shadow, 1px 0 0 0 #909496, 0 0 4px 1px rgba(0, 0, 0, 0.3));
+        outline: var(--terra-switch-slider-focus-outline, 2px dashed #000);
+        outline-offset: var(--terra-switch-slider-focus-outline-offset, 2px);
+      }
+
+      &[data-terra-switch-show-focus-styles='true'] .slider.is-selected:not(.is-disabled) {
+        background-color: var(--terra-switch-slider-selected-focus-background-color, #fff);
+        border: var(--terra-switch-slider-selected-focus-border, 0);
+      }
+    }
+
+    // Removes dotted border of button on focus that shows up in Firefox
+    &::-moz-focus-inner {
```

Comment:
Changing the button to a div will remove the need for this pseudo selector.

Reply:
[removed](https://github.com/cerner/terra-core/pull/2846/commits/893ebe62f9fd9578c95be4985af8c5a9b9545525#diff-323c0506d3b9781e723ca3d110a89c79)